google-cloud-tools-plugin/src/com/google/gct/idea/appengine/util/EndpointUtilities.java

```diff
@@ -75,6 +75,7 @@ public static boolean isEndpointClass(PsiElement psiElement) {
    */
   public static boolean isApiMethod(@NonNls PsiMethod psiMethod) {
     PsiModifierList psiModifierList =  psiMethod.getModifierList();
+    assert psiModifierList != null;
```

Comment:
for this case Preconditions.chechNotNull is more appropriate.  asserts should only be used for things that can't possibly be untrue. In this case the client could conceivably pass a psiMethod that returns a null psiModifierList so an NPE is more appropriate which is what the Preconditions check will throw.